### ISSUE TITLE
CodeMirror indentation unit setting

### ISSUE THREAD

**Issue by fcollonval** [jupyterlab/jupyterlab#14097]
I just noticed that there are some strange defaults set in the code cell settings: Indentation unit is 2 spaces and line numbers are switched on. I'm fine with the line numbers (though it might unnecessarily break user experience) but two-spaces indentation for python is weird.

![image](https://user-images.githubusercontent.com/74381262/221382609-6ccbbe56-6199-492b-a092-ba1445ce4009.png)

I tried this on binder. It might be related to notebook-extension/schema/tracker.json. It seems the settings have been deleted from there?

_Originally posted by @schmidi314 in https://github.com/jupyterlab/jupyterlab/issues/13639#issuecomment-1445220113_

**Comment by fcollonval** [comment]
Good catch, thank you @schmidi314. I would add two observations that can be moved to a separate issue and addressed in a follow up PR:

1. When switching between tab sizing options the "Default:" shown below appears random:
    ![default](https://user-images.githubusercontent.com/5832902/221383802-12b6fed7-8cfc-4baf-9534-2c0e409c7b86.gif)
2. The "Default:" often gets lost for some reason but it does not happen for other groups of settings I tested:
    ![Peek 2023-02-25 23-09](https://user-images.githubusercontent.com/5832902/221383812-84015c50-d4da-46b9-8d71-72df2c7b0564.gif)

_Originally posted by @krassowski in _

**Comment by MMesch** [comment]
The triaging agent has identified this issue as resolved (Rationale below). Looking through it I agree, and I can't reproduce the `Default:` changing randomly or getting lost. It seems to all work fine. If you think the issue is not resolved, please reopen and leave a comment explaining what remains to get fixed!

---

**Rationale**: The issue has been completely resolved by PR #14411 which changed the default indentUnit from '2' to '4' in the CodeMirror extension configuration. Current codebase inspection confirms the default is now '4' spaces, which aligns with Python PEP 8 conventions. The issue can be closed as completed.
